### PR TITLE
Enable GlutenDataFrameWindowFunctionsSuite.

### PR DESCRIPTION
**Pull request by lviiii**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Review by JkSelf — APPROVED** [review]
LGTM

File: backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHValidatorApi.scala [hunk 1]
```diff
@@ -66,25 +66,8 @@ class CHValidatorApi extends IValidatorApi {
    */
   override def doAggregateFunctionValidate(
```
Comment (zzcclp) [inline backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHValidatorApi.scala:67]: Please revert these changes, `CHValidatorApi` doesn;t implement from `GlutenValidatorApi`.

**Review by zzcclp — APPROVED** [review]
LGTM, except one comment.

File: backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHValidatorApi.scala (hunk 1 above)
Comment (lviiii) [reply backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHValidatorApi.scala]: Done
